front/lib/api/llm/clients/openai/utils/conversation_to_openai.ts

```diff
@@ -0,0 +1,187 @@
+import type {
+  FunctionTool,
+  ResponseFunctionToolCallOutputItem,
+  ResponseInput,
+  ResponseInputContent,
+  ResponseInputItem,
+} from "openai/resources/responses/responses";
+import type { ReasoningEffort } from "openai/resources/shared";
+
+import type { AgentActionSpecification } from "@app/lib/actions/types/agent";
+import type {
+  AssistantContentMessageTypeModel,
+  AssistantFunctionCallMessageTypeModel,
+  Content,
+  FunctionMessageTypeModel,
+  UserMessageTypeModel,
+} from "@app/types";
+import type {
+  AgentReasoningEffort,
+  ModelConversationTypeMultiActions,
+} from "@app/types";
+import { isString } from "@app/types";
+import type { AgentContentItemType } from "@app/types/assistant/agent_message_content";
+
+export function toOpenAIReasoningEffort(
+  reasoningEffort: AgentReasoningEffort
+): ReasoningEffort | null {
+  return reasoningEffort === "none"
+    ? null
+    : reasoningEffort === "light"
+      ? "low"
+      : reasoningEffort;
+}
+
+function toUserContent(content: Content): ResponseInputContent {
+  switch (content.type) {
+    case "text":
+      return { type: "input_text", text: content.text };
+    case "image_url":
+      return {
+        type: "input_image",
+        image_url: content.image_url.url,
+        detail: "auto",
+      };
+  }
+}
+
+function toAssistantItem(content: AgentContentItemType): ResponseInputItem {
+  switch (content.type) {
+    case "text_content":
+      return {
+        role: "assistant",
+        type: "message",
+        content: content.value,
+      };
+    case "function_call":
+      return {
+        type: "function_call",
+        id: idFormat(content.value.id),
+        call_id: callIdFormat(content.value.id),
```

Comment:
I'm wondering what's the reason behind removing and adding "call_" ? Why don't we keep it as is and send it back to the model unchanged ?